task/seed.js

```diff
@@ -14,10 +14,10 @@ Promise.all([
     database.sequelize.model('trial').bulkCreate(seedData.trial),
     database.sequelize.model('question_option').bulkCreate(seedData.questionOption),
     database.sequelize.model('question_template').bulkCreate(seedData.questionTemplate),
-    database.sequelize.model('join_questions_and_options').bulkCreate(seedData.joinQuestionsAndOptions),
     database.sequelize.model('survey_template').bulkCreate(seedData.surveyTemplate),
-    database.sequelize.model('join_trials_and_surveys').bulkCreate(seedData.joinTrialsAndSurveys),
-    database.sequelize.model('join_surveys_and_questions').bulkCreate(seedData.joinSurveysAndQuestions)
+    // database.sequelize.model('join_trials_and_surveys').bulkCreate(seedData.joinTrialsAndSurveys),
```

Comment:
@poojaRal `join_trials_and_surveys` is being replaced with `join_stages_and_surveys` do you know which surveys relate to which stage in these trials?